### PR TITLE
CODENVY-224: Enable workspace shutdown wait timeout

### PR DESCRIPTION
**Pull request by akorneta**
@skabashnyuk @evoevodin @sleshchenko @garagatyi

---
This change is [Reviewable](https://reviewable.io/reviews/codenvy/codenvy/23)

### CODE REVIEW

**Comment by riuvshin** [comment]
ci-build

File: core/codenvy-hosted-workspace-activity/src/main/java/com/codenvy/activity/server/WorkspaceActivityService.java [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    @Inject
+    public WorkspaceActivityService(WorkspaceActivityManager workspaceActivityManager) {
+        this.workspaceActivityManager = workspaceActivityManager;
+    }
+
+    @PUT
+    @Path("/{wsId}")
+    @Consumes(APPLICATION_JSON)
+    public void active(@PathParam("wsId") String wsId) {
+        workspaceActivityManager.update(wsId, System.currentTimeMillis());
```
Comment (skabashnyuk) [inline core/codenvy-hosted-workspace-activity/src/main/java/com/codenvy/activity/server/WorkspaceActivityService.java]: We should check here is workspace is running and if user is owner

**Comment by skabashnyuk** [comment]
ок

**Comment by evoevodin** [comment]
ok

**Pull request merged**